src/cmdlib.sh

```diff
@@ -1048,9 +1048,13 @@ cmdlib.write_image_json('${srcfile}', '${outfile}')")
 
 # API to prepare image builds.
 # Ensures that the tmp/repo ostree repo is initialized,
-# and also writes tmp/image.json.
+# and also writes tmp/image.json if arg2 isn't set false.
```

Comment:
Hmm, I think let's just make the argument an integer to make this cross-language goop less yucky than it already is? E.g.

```suggestion
# and also writes tmp/image.json if $2 is unset or 1.
```